app/uk/gov/hmrc/mobiletaxcreditsrenewal/connectors/ShutteringConnector.scala

```diff
@@ -46,7 +46,7 @@ class ShutteringConnector @Inject() (
       .map { json =>
         (json).as[Shuttering]
       } recover {
-      case e: Upstream5xxResponse =>
+      case e: UpstreamErrorResponse =>
```

Comment:
Both the Error cases here essentially do the same thing, so I would just completely remove this one.